packages/edit-post/src/components/layout/style.scss

```diff
@@ -7,14 +7,20 @@
 .edit-post-layout .components-editor-notices__snackbar {
 	position: fixed;
 	right: 0;
-	bottom: 40px;
-	padding-left: 16px;
-	padding-right: 16px;
+	bottom: $grid-unit-50;
+	padding-left: $grid-unit-20;
+	padding-right: $grid-unit-20;
 }
 
 .is-distraction-free {
 	.components-editor-notices__snackbar {
-		bottom: 20px;
+		bottom: calc(#{$grid-unit-50} / 2);
```

Comment:
Same here with the caveat that need to use `math.div` nowadays because SCSS recently deprecated the `/` operator.

```scss
bottom: math.div($grid-unit-50, 2);
```